src/main/java/com/labsynch/labseer/dto/CmpdRegBatchCodeDTO.java

```diff
@@ -105,20 +105,21 @@ public void checkForDependentData() {
 
 	private Collection<ExperimentBatchCodeDTO> findExperimentBatchCodeDTOsFromExperimentValueBatchCodes() {
 		EntityManager em = SubjectValue.entityManager();
-		String sql = "SELECT DISTINCT NEW MAP(p.codeName as protcolCode, e.codeName as experimentCode, el.labelText as experimentName, ev.codeValue as comments, count(ev.id) as description) "
+		String sql = "SELECT DISTINCT NEW MAP(p.codeName as protocolCode, pl.labelText as protocolName, e.codeName as experimentCode, el.labelText as experimentName, ev.codeValue as comments, count(ev.id) as description) "
 				+ "FROM ExperimentValue ev "
 				+ "JOIN ev.lsState as es "
 				+ "JOIN es.experiment as e "
 				+ "LEFT OUTER JOIN e.lsLabels as el "
 				+ "JOIN e.protocol as p "
+				+ "LEFT OUTER JOIN p.lsLabels as pl on pl.ignored = false "
```

Comment:
You'll want to additionally add `and pl.lsKind = 'protocol name' ` to the end of this line